api/DOMRect.json

```
@@ -225,6 +225,104 @@
           }
         }
       },
+      "height": {
+        "__compat": {
+          "spec_url": "https://drafts.fxtf.org/geometry/#dom-domrectreadonly-height",
+          "support": {
+            "chrome": {
+              "version_added": "61"
```

```suggestion
              "version_added": "2"
```

For the properties that were on `ClientRect`, I think we should mark support as going back that far. That's what we do with other prefixed and alt.name. entries.